Src/EB/AMReX_EB2_Level.H

```diff
@@ -479,6 +495,34 @@ GShopLevel<G>::define_fine (G const& gshop, const Geometry& geom,
         }
 
         ParallelAllReduce::Sum<int>({nsmallcells,nmulticuts}, ParallelContext::CommunicatorSub());
+        if (plt_multiple_cuts && nmulticuts > 0)
+        {
+            amrex::Print() << "Total number of multicuts = " << nmulticuts << "\n";
+            amrex::Print() << "plotting multicut locations..." << "\n";
+
+            #if (AMREX_SPACEDIM == 3)
+                if (!multicut_face[0].empty())
+                {
+                    WriteSingleLevelPlotfile("plt.x", multicut_face[0], {"multicut_fcx"}, geom, 0.0, 0);
```

Comment:
Maybe change these files to `plt.multicut.x` and similar so the name if more informative. For the 2D one it shouldn't have the `.x` because there is only one type of multicut in 2D.